test/types/string.js

```diff
@@ -561,7 +551,8 @@ describe('string', () => {
             Helper.validate(schema, [
                 ['joe@example.com', true],
                 ['joe@www.example.com', true],
-                ['joe@localhost', false, null, '"value" must be a valid email'],
+                ['joe@localhost', true],
+                ['joe@', false, null, '"value" must be a valid email'],
```

Comment:
Should we add a test here for a unicode address? `êjness@something.com` for example. We don't want to ensure the operation of `isemail` but I think it would be good to make sure we continue to support unicode going forward.